src/audio/buffer.c

```diff
@@ -174,6 +180,11 @@ void comp_update_buffer_produce(struct comp_buffer *buffer, uint32_t bytes)
 void comp_update_buffer_consume(struct comp_buffer *buffer, uint32_t bytes)
 {
 	uint32_t flags;
+	struct buffer_cb_transact cb_data = {
+		.buffer = buffer,
+		.transaction_amount = bytes,
+		.transaction_begin_address = buffer->r_ptr
```

Comment:
as far as I now understand, this PR will have updates, so you'll have a chance to add commas to struct initialisations like this one

Reply:
done